test/print_test_stats.py

```diff
@@ -42,21 +43,19 @@ def append(self, test_case):
         self.skipped_count += 1 if test_case.skipped else 0
         self.errored_count += 1 if test_case.errored else 0
 
-    def print_report(self):
+    def print_report(self, num_longest):
```

Comment:
nit: for backward compatibility since `print_report` is not `_print_report`
```suggestion
    def print_report(self, num_longest=3):
```